graylog2-server/src/main/java/org/graylog/plugins/views/search/rest/QueryValidationResource.java

```diff
@@ -138,8 +143,18 @@ private ValidationRequest prepareRequest(ValidationRequestDTO validationRequest,
     }
 
     private Set<ExplainResults.IndexRangeResult> indexRanges(ValidationRequest request) {
+        final Set<String> streamTitles = request.streams().stream()
+                .map(streamId -> {
+                    try {
+                        return streamService.load(streamId).getTitle();
+                    } catch (NotFoundException e) {
+                        return null;
+                    }
+                })
+                .collect(Collectors.toSet());
```

Comment:
maybe we should filter out `null` values here too?

Reply:
good point- fixed